lib/test_prof/event_prof/profiler.rb

```diff
@@ -71,17 +73,27 @@ def example_finished(id)
       end
 
       def results
-        {
-          groups: @groups.to_a
+        results = {
+          groups: @groups.to_a,
+          groups_total_time: total_time_for(@groups.to_a)
         }.tap do |data|
           next unless per_example?
 
           data[:examples] = @examples.to_a
+          data[:examples_total_time] = total_time_for(@examples.to_a)
```

Comment:
This is not the "total time" we need; we want to compare the event time with the _absolute_ time (maybe, "total" sounds confusing)

Reply:
Yep